### PR TITLE
Port crm_claim_rma and crm_claim_categ_as_name to v8

### PR DESCRIPTION
**Pull request by aristobulo-eezee**

### CODE REVIEW

File: crm_claim_categ_as_name/__openerp__.py [hunk 1]
```diff
@@ -30,7 +35,7 @@
     """
     Replace claim name by category. It makes easier to filter on claims.
     """,
-    'author': "akretion,Odoo Community Association (OCA)",
+    'author': "Eezee-It",
```
Comment (pedrobaeza) [inline crm_claim_categ_as_name/__openerp__.py]: You cannot replace authorship

**Comment by pedrobaeza** [comment]
Please make one PR per module and sign the CLA

**Comment by nhomar** [comment]
We will have a huge issue here....

@aristobulo-eezee did you see the job already done by Vauxoo here?:

https://github.com/OCA/rma/pull/26

I really appreciate if you give a try because it is hugelly more complete.

But BTW just let me know.

**Comment by aristobulo-eezee** [comment]
Created a separated PR for crm_claim_rma as requested by @pedrobaeza.
You will find it here https://github.com/OCA/rma/pull/32

File: crm_claim_rma/wizard/account_invoice_refund.py [hunk 2]
```diff
@@ -20,27 +21,32 @@
...
-    def _get_description(self, cr, uid, context=None):
-        if context is None:
-            context = {}
         description = context.get('description') or ''
         return description
 
-    _defaults = {
-        'description': _get_description,
-    }
+    description = Char(default=_get_description)
```
Comment (nhomar) [inline crm_claim_rma/wizard/account_invoice_refund.py:40]: Can you use fields.Char, I kon it save space but at the end migrations are more difficult.

Comment (aristobulo-eezee) [inline crm_claim_rma/wizard/account_invoice_refund.py:40]: you are looking the wrong PR it's #32

**Pull request closed without merging**